h2o-core/src/main/java/water/fvec/Frame.java

```diff
@@ -859,7 +859,7 @@ public Frame extractFrame(int startIdx, int endIdx) {
 
   public Frame remove( String[] names ) {
     for( String name : names )
-      remove(find(name));
```

Comment:
Vec shouldn't be deleted, purpose of the remove method is to remove them from the Frame

Reply:
Ok, it was not clear for me based on naming and method's comments. Thanks!